Sources/Aardvark/BugReporting/ARKBugReportAttachment.swift

```diff
@@ -0,0 +1,46 @@
+//
+//  Copyright 2021 Square, Inc.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+
+import Foundation
+
+public final class ARKBugReportAttachment: NSObject {
+
+    // MARK: - Life Cycle
```

Comment:
absolute nit:
```suggestion
    // MARK: - Lifecycle
```

Reply:
Or actually, didn't the Square style guide used to stipulate:
```suggestion
    // MARK: - Initialization
```

Reply:
We updated our style guide to use `Life Cycle` a few years ago.